src/main/java/cz/muni/ics/oidc/props/AttrsMapping.java

```diff
@@ -57,6 +59,7 @@
                         masterProxyIdentifier, proxyIdentifier, isTestSp, isOidc, managersGroupId)
         );
         attrNames.addAll(contacts);
+        attrNames.addAll(homePageUris);
```

Comment:
Probably better to check if the passed argument is not null, as in such a case the thing would break.